src/style.css

```
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  width: 140px;
+  justify-content: space-evenly;
+  margin: auto;
+}
+
+.stars--0 .stars__star,
+
+.stars--1 .stars__star,
+
+.stars--2 .stars__star,
+
+.stars--3 .stars__star,
+
+.stars--4 .stars__star,
+
+.stars--5 .stars__star {
```

```suggestion
.stars__star {
```
Стилізуй просто клас .stars__star